packages/zoe/src/cleanProposal.js

```diff
@@ -15,76 +15,98 @@ import { arrayToObj } from './objArrayConversion';
 // `{ waived: null }` `{ onDemand: null }` `{ afterDeadline: { timer
 // :Timer, deadline :Number } }
 
-// Assert that the keys of record, if present, are in expectedKeys.
-// Return the keys after asserting this.
-const checkKeys = (expectedKeys, record) => {
-  // Assert that keys, if present, match expectedKeys.
+export const assertCapASCII = keyword => {
```

Comment:
cleanProposal.js is exporting a few things that aren't covered by the current name. Would it be better to rename it, or to split its functions up further?

The comment above no longer covers the file, so it should be moved closer to the function it describes. (Or the fn moved up here if it doesn't depend on the other fns.)

Reply:
Yeah, I'm not sure how to split up the content of this file versus `objArrayConversion.js`. They are both more generalized helpers now. 

I will move that comment!